### PR TITLE
Update executable template

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description
- update executable template
- update doc/sample
- trace call cli pfs to support detch mode in executable/msi

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow-devkit/promptflow/_sdk/data/executable/start_pfs.vbs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DIM objshell
```
Comment (wangchao1230) [inline src/promptflow-devkit/promptflow/_sdk/data/executable/start_pfs.vbs:1]: do we copied installer scripts here? shall this be a move?

Comment (YingChen1996) [reply src/promptflow-devkit/promptflow/_sdk/data/executable/start_pfs.vbs:1]: yes, copy here to support executable run pfs in detachmode when tracing

File: src/promptflow-devkit/promptflow/_sdk/data/executable/app.spec.jinja2 [hunk 2]
```diff
@@ -29,26 +51,34 @@ a = Analysis(
...
     console=True,
     disable_windowed_traceback=False,
     argv_emulation=False,
     target_arch=None,
     codesign_identity=None,
     entitlements_file=None,
+    contents_directory='.',
+)
+
+coll = COLLECT(
```
[the remark this answers is not in the document]
Comment (YingChen1996) [reply src/promptflow-devkit/promptflow/_sdk/data/executable/app.spec.jinja2:75]: one-folder mode here to debug， will change to one file mode when all test passed

[the remark this answers is not in the document]
Comment (YingChen1996) [reply src/promptflow-devkit/promptflow/_sdk/data/executable/app.spec.jinja2]: changed to on file

**Review by wangchao1230 — APPROVED** [review]

**Review by brynn-code — APPROVED** [review]

**Pull request merged**